### PR TITLE
Add contributor docs and an example import using Template MCF + StatVarObservation.

### PR DESCRIPTION
**Pull request by tjann**
Note: Scripting Instance MCF might make more sense for data that needs to be refreshed daily. In this case, we could use the preprocessing script as the download process. So the preprocessing would need cron-ing.

### CODE REVIEW

File: docs/mcf_format.md [hunk 1]
```diff
@@ -0,0 +1,321 @@
...
+hasResponseOption: E:SomeDataset->E1
+typeOf: SurveyItem
+dcid: C:SomeDataset->SurveyItem_Dcid
+```
+
+Where E is short for "Entity" and "C" is short for "Column".
+
+Contributors need not worry about the process of using template MCF to convert
+datasets to instance MCF, but roughly, for each row of the dataset, we create
+all the nodes in the template MCF specification, subbing in values from the
```
Comment (pradh) [inline docs/mcf_format.md:319]: s/subbing in/substituting/

File: scripts/covid_tracking_project/historic_state_data/preprocess_csv.py [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+      writer.writerow(processed_dict)
+
+# Automate Template MCF generation since there are 18 Statitical Variables.
+
+TEMPLATE_MCF_TEMPLATE = """
+Node: E:COVIDTracking_States->E{index}
+typeOf: StatVarObservation
+variableMeasured: {stat_var}
+observationAbout: C:COVIDTracking_States->GeoId
+observationDate: C:COVIDTracking_States->Date
```
Comment (pradh) [inline scripts/covid_tracking_project/historic_state_data/preprocess_csv.py:53]: quotes?

File: scripts/covid_tracking_project/historic_state_data/preprocess_csv.py [hunk 3]
```diff
@@ -0,0 +1,60 @@
+import csv
```
Comment (pradh) [inline scripts/covid_tracking_project/historic_state_data/preprocess_csv.py:1]: (later is fine) might be good if the example has a test. that way, there is some chance that contributors will provide one.

File: docs/representing_statistics.md [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+observationAbout: dcid:geoId/4865000
+observationDate: "2014"
+value: 39.4
+```
+
+Where [`MedianAge`](https://browser.datacommons.org/kg?dcid=MedianAge)
+is a `StatisticalVariable` schema node that only needs to be defined once.
+
+
+## Mapping `StatisticalVariable` and `StatVarObservation` to `StatisticalPopulation` and `Observation`
```
Comment (pradh) [inline docs/representing_statistics.md:62]: The content here sort presupposes user understands StatPop/Obs. Should we link in the data-model doc that delves into those? Like the one that I think Antares curated.

File: docs/representing_statistics.md [hunk 5]
```diff
@@ -0,0 +1,127 @@
...
+Node: StatisticalPopulation_People_SanAntonio_TX
+typeOf: dcid:StatisticalPopulation
+populationType: dcid:Person
+location: dcid:geoId/4865000
+```
+
+The `Observation` copies the `StatisticalVariable`'s `measuredProperty`,
+`measurementMethod`, `measurementQualifier`, `unit`, etc.
+and the `StatVarObservation`'s `observationDate`. It also extracts the
+`statType`value from the `StatisticalVariable` and the `value` value from
```
Comment (edumorlom) [inline docs/representing_statistics.md:95]: should there be space after `statType` and before value?

File: docs/representing_statistics.md [hunk 6]
```diff
@@ -0,0 +1,127 @@
...
+[Schema.org Data Model](https://schema.org/docs/datamodel.html),
+and its schema is a superset of
+[Schema.org schema](https://schema.org/docs/schemas.html).
+
+The most commonly used high-level schema types for representing statistical data are:
+
+- [`Class`](https://schema.org/Class)
+- [`Property`](https://schema.org/Property)
+- [`StatisticalPopulation`](https://schema.org/StatisticalPopulation)
+- [`Observation`](https://schema.org/`Observation`)
```
Comment (edumorlom) [inline docs/representing_statistics.md:17]: I think you didn't mean to put the ` in the url. I'm getting a 404 error

File: docs/representing_statistics.md [hunk 7]
```diff
@@ -0,0 +1,127 @@
...
+typeOf: dcid:StatisticalPopulation
+populationType: dcid:Person
+location: dcid:geoId/4865000
+```
+
+The `Observation` copies the `StatisticalVariable`'s `measuredProperty`,
+`measurementMethod`, `measurementQualifier`, `unit`, etc.
+and the `StatVarObservation`'s `observationDate`. It also extracts the
+`statType`value from the `StatisticalVariable` and the `value` value from
+the `StatVarObservation` as its own `.*Value` property and value.
```
Comment (edumorlom) [inline docs/representing_statistics.md:96]: This last paragraph is a bit confusing, too many **values** in my opinion.

File: docs/mcf_format.md [hunk 8]
```diff
@@ -0,0 +1,321 @@
...
+```
+
+To refer to an entity via its identifier (`id1`) in a namespace associated with
+a prefix 'foaf', we write 'foaf:id1'. To refer to an entity via its identifier
+in the local namespace, we use the prefix `l`.
+
+
+#### Special Namespaces
+
+The Data Commons graph is built on top of schema.org vocabulary and its Data
```
Comment (edumorlom) [inline docs/mcf_format.md:91]: Sometimes you use Schema.org and other times schema.org

File: scripts/covid_tracking_project/historic_state_data/preprocess_csv.py (hunk 2 above)
Comment (tjann) [reply scripts/covid_tracking_project/historic_state_data/preprocess_csv.py:53]: Seems table2mcf takes care of the quoting. Actually doesn't work if I add quotes to observationDate. I did add in dcs: to typeOf, variableMeasured.

File: docs/representing_statistics.md (hunk 7 above)
Comment (tjann) [reply docs/representing_statistics.md:96]: Did a little shuffling, hopefully this helps.

File: docs/representing_statistics.md (hunk 4 above)
Comment (tjann) [reply docs/representing_statistics.md:62]: I will do this and the test in a separate PR.

**Pull request merged**